tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/TinkerGraphProvider.java

```diff
@@ -107,7 +107,7 @@ protected static boolean requiresPersistence(final Class<?> test, final String t
      */
     protected static boolean requiresListCardinalityAsDefault(final LoadGraphWith.GraphData loadGraphWith,
                                                             final Class<?> test, final String testMethodName) {
-        return loadGraphWith == LoadGraphWith.GraphData.CREW
+        return loadGraphWith == LoadGraphWith.GraphData.CREW && !testMethodName.startsWith("shouldReadWriteCrew")
```

Comment:
We don't set default vertex property cardinality as LIST for `shouldReadWriteCrew` test case. By doing so, `shouldReadWriteCrew` will fail without the changes in this PR because TinkerPop will use SINGLE cardinality by default.